perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -230,7 +234,26 @@ def _Delete(self):
         'terminate-instances',
         '--region=%s' % self.region,
         '--instance-ids=%s' % self.id]
-    vm_util.IssueRetryableCommand(delete_cmd)
+    vm_util.IssueCommand(delete_cmd)
+
+  def _Exists(self):
+    """Returns true if the VM exists."""
+    describe_cmd = util.AWS_PREFIX + [
+        'ec2',
+        'describe-instances',
+        '--region=%s' % self.region,
+        '--filter=Name=instance-id,Values=%s' % self.id]
+    stdout, _ = vm_util.IssueRetryableCommand(describe_cmd)
+    response = json.loads(stdout)
+    reservations = response['Reservations']
+    assert len(reservations) < 2, 'Too many reservations.'
+    if not reservations:
+      return False
+    instances = reservations[0]['Instances']
+    assert len(instances) == 1, 'Wrong number of instances.'
+    status = instances[0]['State']['Name']
+    assert status in INSTANCE_KNOWN_STATUSES
```

Comment:
Could you add the status here (`assert status in INSTANCE_KNOWN_STATUSES, status`), so the erroneous status is shown in the error message? Could be helpful for debugging.